internal/crypto/Cargo.toml

```diff
@@ -63,9 +63,26 @@ default-features = false
 features = ["now", "wasmbind"]
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
+ecdsa = "0.16.9"
+ed25519-dalek = "2.1.1"
+p256 = "0.13.2"
+p384 = "0.13.0"
+rsa = { version = "0.9.6", features = ["sha2"] }
+spki = "0.7.3"
 wasm-bindgen = "0.2.83"
```

Comment:
wasm-bindgen, web-sys, web-time can not be included in the wasi target. I know this PR isn't really about adding WASI support, but there are other wasi targets in the Cargo.toml